### PR TITLE
Health bar skin files organization

### PR DESCRIPTION
**Pull request by stefaniamak**
Related PR
* https://github.com/Terasology/LightAndShadow/pull/124

# Result

We now have 1 skin file (`healthHud.skin`) to change the size of the icons within one file, which is located within the LightAndShadow module. The rest 3 skin files located in the LightAndShadowResources module inherit the size from the mentioned skin file.

## Changes

* Deleted all size attributes within the 3 skin files within LightAndShadowResources
* Made them inherit the size within the LightAndShadow skin about the health bar.

The file from which they inherit from can be found through this draft PR: https://github.com/Terasology/LightAndShadow/pull/124.

## How to test

Make sure you have set up the https://github.com/Terasology/LightAndShadow/pull/124 as well.

* Start a LaS game (both old or new game file works fine)
* Check the size of the health bar
* You can enter a team by clicking one of the two (black or red) boxes, to check the size of the other teams' icon size.

Extra: you can go ahead and change the size number placed within the `healthHud.skin` from the lines:
```json
"icon": {
    "fixed-width": 30,
    "fixed-height": 30
}
```
to make extra sure.

### CODE REVIEW

**Comment by jdrueckert** [comment]
Uhmm... Maybe I misunderstood the PR description, but...
> The rest 3 skin files located in the LightAndShadowResources module inherit the size from the mentioned skin file.

Isn't that exactly the LASR -> LAS dependency we didn't want to have? :thinking:
Thought we're going for LAS -> LASR... but maybe I'm mixing something up?

**Comment by stefaniamak** [comment]
>  that exactly the LASR -> LAS dependency we didn't want to have? 🤔

Yes, it is.. Skaldarnar mentioned this as well. I plan on moving all health realared UI/skin files within LaSR.

File: assets/skins/healthHud.skin [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                "UIIconBar": {
+                    "parts": {
+                        "icon": {
+                            "background": "CoreAssets:icons#darkHeartOutline"
+                        }
+                    }
+                }
+            }
+        }
+    }
```
Comment (jdrueckert) [inline assets/skins/healthHud.skin]: can be removed as healthHud is only used as a base to inherit from

File: assets/skins/healthBlack.skin [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "inherit": "default",
+    "inherit": "LightAndShadowResources:healthHud",
     "elements": {
```
Comment (jdrueckert) [inline assets/skins/healthBlack.skin]: can be removed as it's inherited from healthHud

File: assets/skins/healthRed.skin [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-    "inherit": "default",
+    "inherit": "LightAndShadowResources:healthHud",
     "elements": {
```
Comment (jdrueckert) [inline assets/skins/healthRed.skin]: can be removed as it's inherited from healthHud

File: assets/skins/healthWhite.skin [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-    "inherit": "default",
+    "inherit": "LightAndShadowResources:healthHud",
     "elements": {
```
Comment (jdrueckert) [inline assets/skins/healthWhite.skin]: can be removed as it's inherited from healthHud

**Review by jdrueckert — CHANGES_REQUESTED** [review]
Guess these changes were lost when moving the files, I believe you had them integrated earlier in the LAS PR (124).
While we're on it, let's also rename `healthHud.skin` to `healthBase.skin` to make it clear, that this is the base for red, black and white to inherit from.

**Pull request merged**